base/etc/nginx/conf.d/location/10-block.conf

```diff
@@ -2,8 +2,28 @@ location ~ (^|/)\. {
   return 403;
 }
 
-# Protect files and directories from prying eyes.
-location ~* \.(log|engine|inc|install|make|module|profile|po|sh|.*sql|theme|twig|tpl(\.php)?|xtmpl|yml)(~|\.sw[op]|\.bak|\.orig|\.save)?$|/(\.(?!well-known).*|Entries.*|Repository|Root|Tag|Template|composer\.(json|lock)|web\.config)$|/#.*#$|\.php(~|\.sw[op]|\.bak|\.orig|\.save)$ {
+# Protect Drupal-specific files and directories from prying eyes.
```

Comment:
I would like to keep the rules consistent with what is in the Drupal .htaccess file.

That will allow us to better maintain and keep this up to date with core.

I would only like things to be split out which are not in the Drupal .htaccess file.

Reply:
Updated and added a before and after to the PR description to show the change enforced.